### PR TITLE
Documents workflow for private and shared notes

### PR DESCRIPTION
**Pull request by ccamara**
Context: after [this discussion in discord](https://discord.com/channels/717965437182410783/917091167932383344/917129436250390619), I thought it could be a good idea to integrate @kevins8 's gist https://gist.github.com/kevinslin/0e0f13fedb43732e86938ab1033b7efd into a note within dendron.

I've created a new workflow guide which is not anything but a step-by-step use of a multi-vault strategy. Feel free to integrate if you can see value here or simply ignore it (it was useful for me, at least, as I learned how to do it by writing it).

### CODE REVIEW

**Comment by ccamara** [comment]
Not sure if the approach I'm sharing here is the best possible one. Comments are welcome.

File: vault/dendron.guides.workflows.private-shared-setup.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+   3. Select `blank`
+2. Add as many [[dendron.topic.vaults]] as needed, each one serving different purposes or audiences.
+   
+   1. Create a **local note** for notes that will not be shared ([[more info here|dendron.topic.vaults#vault-add]]):
+      1. Run `Dendron: Vault Add`
+      2. Select `local`
+      3. Type the location where you want it to be stored
+      4. Type the name of the vault (can be different from the folder's name)
+   2. Add a **remote vault** for notes that will be shared (either with yourself or with others) you can either clone a remote note or create a local vault and then convert it into a remote one:
+      1. Adding a remote vault:
```
Comment (ScriptAutomate) [inline vault/dendron.guides.workflows.private-shared-setup.md:38]:
```suggestion
      1. If adding remote vault(s) that already exist: 
```

File: vault/dendron.guides.workflows.private-shared-setup.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      2. Select `local`
+      3. Type the location where you want it to be stored
+      4. Type the name of the vault (can be different from the folder's name)
+   2. Add a **remote vault** for notes that will be shared (either with yourself or with others) you can either clone a remote note or create a local vault and then convert it into a remote one:
+      1. Adding a remote vault: 
+         1. Run `Dendron: Vault Add`
+         2. Select `remote`
+         3. Write the git location of the remote vault you want to clone
+         4. The vault will be cloned inside your workspace
+      2. Converting a local vault into a remote vault:
```
Comment (ScriptAutomate) [inline vault/dendron.guides.workflows.private-shared-setup.md:43]:
```suggestion
      2. If converting local vault(s) into remote vault(s):
```

File: vault/dendron.guides.workflows.private-shared-setup.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+   2. Type the location where you want it to be stored
+   3. Select `blank`
+2. Add as many [[dendron.topic.vaults]] as needed, each one serving different purposes or audiences.
+   
+   1. Create a **local note** for notes that will not be shared ([[more info here|dendron.topic.vaults#vault-add]]):
+      1. Run `Dendron: Vault Add`
+      2. Select `local`
+      3. Type the location where you want it to be stored
+      4. Type the name of the vault (can be different from the folder's name)
+   2. Add a **remote vault** for notes that will be shared (either with yourself or with others) you can either clone a remote note or create a local vault and then convert it into a remote one:
```
Comment (ScriptAutomate) [inline vault/dendron.guides.workflows.private-shared-setup.md:37]:
```suggestion
   2. Add a **remote vault** for notes that will be shared (either with yourself or with others). You can clone a remote vault or create a local vault that you convert into a remote one:
```

File: vault/dendron.guides.workflows.private-shared-setup.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+You want to use dendron for personal and professional use, keeping some notes private, while sharing others with other team members (or even yourself! -i.e. different machines). Some of these notes, in turn, may be [[published|dendron.topic.publish]], whereas others, not.
+
+Some specific use cases in which following a [[dendron.topic.multi-vault]] strategy would be recommended are:
+
+![[dendron.topic.multi-vault#use-cases,1:#*]]
+
+## Overview
+
+Create a blank worspace that will have [[multiple vaults|dendron.topic.multi-vault]], each of them serving different purposes and audiences. Some of them, or even all of them can be independent git repositories that can be shared with others as you would with a regular git repo. The workspace, in turn, can also be a git repo itself.
```
Comment (ScriptAutomate) [inline vault/dendron.guides.workflows.private-shared-setup.md:19]:
```suggestion
Create a blank worspace that will have [[multiple vaults|dendron.topic.multi-vault]], each serving different purposes and audiences. Each vault can be independent git repositories that can be shared with others as you would with a regular git repository. The workspace, itself, can also be a git repository.
```

**Review by ScriptAutomate — APPROVED** [review]

**Comment by ScriptAutomate** [comment]
Thank you for the contribution! :partying_face: :seedling:

**Comment by ccamara** [comment]
To you for creating Dendron!

**Pull request merged**